addon/components/fa-icon.js

```diff
@@ -16,8 +16,8 @@ function classList () {
     'fa-fw': this.get('fixedWidth'),
     'fa-border': this.get('border'),
     'fa-li': this.get('listItem'),
-    'fa-flip-horizontal': this.get('flip') === 'horizontal' || this.get('flip') === 'both',
-    'fa-flip-vertical': this.get('flip') === 'vertical' || this.get('flip') === 'both',
```

Comment:
So correct me if I'm wrong but this will be a backward-incompatible change, yes?

Reply:
Yes. I wasn't able to see how to support the `flip=both`. It might be possible, but I figured this early in the game some BC break is ok to simplify the API and align with fontawesome itself. We could ignore this property too, but since it is done with simple CSS it seems a shame to not transform it.